### ISSUE TITLE
Wrong mergedjars directory

### ISSUE THREAD

**Issue by Burtan** [beryx/badass-jlink-plugin#2]
Hi,
I've taken a look at the code and it seems that the mergedjars dir is incorrect.

mergedjars:
```
mergedJarsDir.set(project.layout.buildDirectory.dir(PathUtil.getMergedJarsDirPath(jlinkBasePath.get())))
```
delegatedModulesDir
```delegatedModulesDir.set(new File(PathUtil.getDelegatedModulesDirPath(jlinkBasePath.get())))```

nonModularJarsDir
```nonModularJarsDir.set(new File(PathUtil.getNonModularJarsDirPath(jlinkBasePath.get())))```

**Comment by siordache** [comment]
I'm not able to reproduce your problem. Can you make your project available in a public repository?
If this is not possible, I need at least your build.gradle file and the output of running gradle with `--info --stacktrace`.
But, of course, making your project available in a public repository would be much, much better.

**Comment by siordache** [comment]
@Burtan Do you still get this error?
If yes, can you provide more details to help me reproduce it?

**Comment by Burtan** [comment]
I think it has to do with kotlin. I'll post some more details or a small sample on the weekends!

**Comment by Burtan** [comment]
I created a pull request with a small kotlin test project

**Comment by siordache** [comment]
Fixed in release 1.4.4.
